__init__.py

```diff
@@ -178,9 +162,12 @@ def update_skill_datetime(self):
         this to determine the last time MSM updated any skills.
         """
         if self.is_development_device:
-            skills_dir = Path("/opt/mycroft/skills")
-            last_skill_update = _find_latest_skill_update(skills_dir)
-            self.gui["skillDateTime"] = last_skill_update
+            skills_repo = Git("/opt/mycroft/.skills-repo")
```

Comment:
<kuch> xdg <kuch>

Reply:
Ouch hardcoded skill paths, yeah that's bad. Is there not a method in mycroft somewhere to just return the skills repo?

Reply:
It is only used for development purposes, but it is about the habbit of not using hardcoded paths as users could have changed it (the path is already within mycroft.conf even without XDG fully merged)

check the mycroft.conf. There is a data_folder setting for it.

Reply:
Yeah good spotting - addressed in PR #11